src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/AuthCodeRequestComponent.cs

```diff
@@ -132,19 +132,19 @@ public Uri GetAuthorizationUriWithoutPkce()
 
         private Dictionary<string, string> CreateAuthorizationRequestParameters(Uri redirectUriOverride = null)
         {
-            var extraScopesToConsent = new SortedSet<string>();
+            var extraScopesToConsent = new HashSet<string>();
```

Comment:
Suggest to add comparare if case and cultural doesn't matter, so add StringComparer.InvariantCultureIgnoreCase